web.config

```diff
@@ -7,7 +7,7 @@
   <httpPlatform stdoutLogEnabled="true"
 stdoutLogFile="%HOME%\LogFiles\stdout"
 processPath="%JAVA_HOME%\bin\java.exe"
-arguments="-javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.5.1.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\bulk-scanning-payment-api.jar&quot;">
+arguments="-javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-3.6.2.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\bulk-scanning-payment-api.jar&quot;">
```

Comment:
Don't know if it is still used or not, but you can remove `-javaagent` option, as it is set by base java image, and it takes app insights version from Dockerfile. Basically removing from here shouldn't change anything, but make one place less to look after

Reply:
java agent hasn't been updated in years too, so doubt this does anything